.eslintrc.json

```diff
@@ -18,8 +18,7 @@
     "plugin:eslint-comments/recommended",
     "plugin:@typescript-eslint/recommended",
     "plugin:import/typescript",
-    "prettier",
-    "prettier/@typescript-eslint"
```

Comment:
Could you please elaborate on why "prettier/@typescript-eslint" is no more needed in .eslitrc?

Reply:
It's now part of `@typescript-eslint/recommended`